docs/scenarios_authoring/Intro.md

```diff
@@ -17,7 +17,9 @@ Nussknacker nodes process events; once the node finishes processing of the event
 Typically events processed by Nussknacker come from Kafka topics;  Nussknacker source components are used to read events from Kafka topic and inject them into Nussknacker scenarios. 
 There are cases though when a node can produce a new event, a Tumbling-window or and Session-window component being good examples. 
 
-## (Streaming-Flink only) Notion of time
+## Streaming-Flink
```

Comment:
Streaming-Flink -> as an exception, because this is a section title, I would write Flink engine only